pkgs/data/themes/alacritty-theme/default.nix

```
@@ -26,9 +26,7 @@ stdenvNoCC.mkDerivation (self: {
     runHook postInstall
   '';
 
-  passthru.updateScript = nix-update-script {
-    extraArgs = [ "--version=branch" ];
-  };
+  passthru.updateScript = unstableGitUpdater {};
```

```suggestion
  passthru.updateScript = unstableGitUpdater {
    hardcodeZeroVersion = true;
  };
```

Mind rebasing and setting this? Just merged  #280501 which adds this option, and should also fix the version scheme to use the `0-` prefix.